Classes/Utility/ContentUtility.php

```diff
@@ -1,5 +1,6 @@
 <?php
-declare(strict_types = 1);
+
+declare(strict_types=1);
```

Comment:
which rule from PSR-12 applies here? in TYPO3 Core it's formatted with spaces around "="

Reply:
My checks went entirely after PSR12, no additional special configuration
 

> Declare statements MUST contain no spaces and MUST be exactly declare(strict_types=1) (with an optional semi-colon terminator).
https://www.php-fig.org/psr/psr-12/#3-declare-statements-namespace-and-import-statements

Reply:
you're right!